shared_model/validators/validators_common.hpp

```diff
@@ -36,6 +37,14 @@ namespace shared_model {
        * - BlockLoader
        */
       const bool txs_duplicates_allowed;
+
+      /**
+       * Parameter specifying transaction how many hours in the past
+       * since current peer time transaction's `created_time` can be setted.
+       * The default value is FieldValidator::kDefaultMaxDelay hours.
+       * The value should be the same in a whole iroha network.
+       */
```

Comment:
```suggestion
      /**
       * A parameter, which specifies how many hours before the current peer's `created_time` can the transaction be set. 
       * Default is `FieldValidator::KDefaultMaxDelay` (hours). 
       * The value must be synchronised across all peers.  
       */
```

Reply:
This is from my changes, so I'll accept Your suggestion:)

Reply:
Corrected in https://github.com/hyperledger/iroha/pull/3328/

Reply:
https://github.com/hyperledger/iroha/pull/3328/ was merged, so resolved